src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -53,14 +59,24 @@ public static ParenthesizedExpressionSyntax PlaceWithConditionalExpression(Expre
         {
             // place the mutated statement inside the if statement
             return SyntaxFactory.ParenthesizedExpression(
-                SyntaxFactory.ConditionalExpression(
-                condition: GetBinaryExpression(mutantId),
-                whenTrue: mutated,
-                whenFalse: original))
+                    SyntaxFactory.ConditionalExpression(
+                        condition: GetBinaryExpression(mutantId),
+                        whenTrue: mutated,
+                        whenFalse: original))
                 // Mark this node as a MutationConditional node. Store the MutantId in the annotation to retrace the mutant later
                 .WithAdditionalAnnotations(new SyntaxAnnotation(MutationConditional, mutantId.ToString()));
         }
 
+        public static ExpressionSyntax FindOriginalNonMutatedNode(ExpressionSyntax node)
+        {
+            while (node.GetAnnotations(MutationConditional).Any())
```

Comment:
Should this not also check for MutationIf? It seems like currently this only finds mutations placed with conditionals. Is that intended?

Reply:
Actually, this method is no longer used ==> removed